SPECS/tuned/tuned.spec

```diff
@@ -36,7 +36,7 @@ Requires: python3-configobj
 Requires: python3-dbus 
 Requires: python3-decorator
 Requires: python3-linux-procfs
-Requires: python3-perf
+Requires: (python3-perf or python3-perf-rt)
```

Comment:
@dmcilvaney what is our graph's behavior for this kind of "or" for one package built in specs and another built in specs-extended, but we're only building our core specs?

Reply:
I'll double check, but I believe it takes the pessimistic approach and ensures both are built. I'll need to figure out what the scheduler picks though.

Reply:
I queued https://dev.azure.com/mariner-org/mariner/_build/results?buildId=773396&view=results to double check experimentally as well.

Reply:
I also queued off a buddy build for `tuned` and it does have an "unresolved dependency" failure on python3-perf-rt missing, even though `tuned` went ahead and was built.

https://dev.azure.com/mariner-org/mariner/_build/results?buildId=773389&view=logs&j=bac295d9-f2e0-5165-e63e-e76f383e1a27&t=11f6dde0-e75e-52b2-fbf0-d83a00678fa3